src/panoptes/pocs/core.py

```diff
@@ -435,7 +435,7 @@ def has_free_space(self, directory=None, required_space=0.25 * u.gigabyte, low_s
         """
         directory = directory or os.getenv('PANDIR')
         req_space = required_space.to(u.gigabyte)
-        self._free_space = get_free_space(dir=directory)
+        self._free_space = get_free_space(directory=directory)
```

Comment:
Thanks. Note that as of right now this is `panoptes-utils v0.2.22dev` (via github) whereas the pip install is currently still at `v0.2.21` and that is expecting `dir`. I'll try to get a release out soon.

Reply:
Just a note that `panoptes-utils==0.2.22` has been released and the requirements for POCS have been bumped (in #992).